web/packages/teleport/src/DesktopSession/useTdpClientCanvas.tsx

```diff
@@ -69,16 +67,16 @@ export default function useTdpClientCanvas(props: Props) {
   const initialTdpConnectionSucceeded = useRef(false);
   const encoder = useRef(new TextEncoder());
   const latestClipboardDigest = useRef('');
+  const keyboardHandler = useRef(new KeyboardHandler());
```

Comment:
We should be able to drop the initialization of `KeyboardHandler` here since it's already done in the effect.

Reply:
I tried but couldn't get the type system to play nicely with it. I could make this a `MutableRefObject<KeyboardHandler | undefined>` but that's misleading, because in effect its always there when it matters.

Reply:
FWIW, `const keyboardHandler = useRef<KeyboardHandler>();` doesn't yield any type errors for me.